src/query/block/information.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package block
+
+func (t BlockType) String() string {
+	switch t {
+	case BlockM3TSZCompressed:
+		return "M3TSZ_compressed"
+	case BlockDecompressed:
+		return "decompressed"
+	case BlockScalar:
+		return "scalar"
+	case BlockLazy:
+		return "lazy"
+	case BlockTime:
+		return "time"
+	case BlockContainer:
+		return "container"
+	case BlockWrapper:
+		return "wrapper"
+	case BlockConsolidated:
+		return "consolidated"
+	}
+
+	return "unknown"
+}
+
+type BlockInformation struct {
```

Comment:
Needs comment yeah? (think the exported types in this file are all missing comments)

Reply:
Ah fair enough; thought the general idea was functions and types have full names but variables have Go-ish names, will change it